### ISSUE TITLE
Installing via gradle

### ISSUE THREAD

**Issue by dneykov** [azavea/AndroidValidatedForms#30]
Hi,
I was using original repo for my project but this looks better with more functionalities including ImageController etc.
I'm consider switching to this repo. Can I install it via gradle?

Thanks.

**Comment by flibbertigibbet** [comment]
Glad to hear you're interested in this project!

This fork was created to support form validation via Hibernate, which you may not need for your purposes; Hibernate and its related packages are large dependencies.

The project is distributed on [JitPack](https://jitpack.io/). It can be installed with gradle by adding Jitpack to the `repositories` under `buildscript`: `maven { url "https://jitpack.io" }`, then adding the package under dependencies: `compile 'com.github.azavea:AndroidValidatedForms:1.2.1'`.

**Comment by dneykov** [comment]
@flibbertigibbet 
So if I will use only the form builder there will be lots of unused dependencies? But I can use Hibernate  for validating the form right?
I'm not familiar with Hibernate for what can be used?

Thanks for your time.

**Comment by flibbertigibbet** [comment]
Right, if you're not going to be using the Hibernate Validator annotations, there will be lots of unused dependencies. For examples of using the Hibernate validation annotations on a model, you can take a look at [the sample app model](https://github.com/azavea/AndroidValidatedForms/blob/develop/sample/src/main/java/com/azavea/androidvalidatedforms/sample/TestModel.java).

[Here](http://hibernate.org/validator/documentation/getting-started/#applying-constraints) is some Hibernate Validator documentation on how the annotations work.

The existing Android forms project also has its own validation hooks that will let you write your own validation methods (without needing Hibernate).

**Comment by dneykov** [comment]
Thanks for explanation. I can't use annotation validation because I'm building dynamic form which comes from API.
I'm gonna use your implementation for ImageController.

**Comment by maheshpndt** [comment]
Hi,
I want to use this library implementation for dynamic form but face some issue ( related to update ) after adding it to my project not support to gradle:4.0.0
